.github/workflows/trigger_release_new.yml

```diff
@@ -22,6 +30,11 @@ env:
   TURBO_VERSION: 2.3.3
   NODE_LTS_VERSION: 20
 
+# Limit the permissions of GITHUB_TOKEN for this
+# workflow which is good security practice.
```

Comment:
Comment is not really explaining much. It should explain what each permission is needed for instead.